dftimewolf/cli/dftimewolf_recipes.py

```diff
@@ -292,6 +292,20 @@ def CleanUpPreflights(self) -> None:
     """Calls the preflight's CleanUp functions."""
     self._state.CleanUpPreflights()
 
+  def PrintStats(self) -> None:
+    """Prints collected stats if existing."""
+    statentries = self._state.GetStats()
+    if not statentries:
+      logger.info('No statistics collected during execution.')
+
+    logger.info(f'{len(statentries)} stat entries collected during execution.')
+    for entry in statentries:
+      logger.debug(f'[{entry.module_name} ({entry.module_type})] {entry.stats}')
+
+  def ExportStats(self) -> None:
+    """Exports collected stats if existing. Default behavior is to log."""
+    self.PrintStats()
```

Comment:
Is there a plan to extend this to publish stats to a file, or external service for collection?

Reply:
Yes absolutely, and they should override this very `ExportStats` method